tests/modules/services/window-managers/i3/i3-followmouse-expected.conf

```diff
@@ -104,3 +104,4 @@ bar {
 
 
 
+
```

Comment:
```suggestion
```

Reply:
Doesn't seem a newline was missing at the end of file.

Reply:
There is a general problem, that i3/sway config-generation works different.

https://github.com/nix-community/home-manager/blob/2dc090fd7f7e0a76cff87c0e68ae15118e60e327/modules/services/window-managers/i3-sway/i3.nix#L189
https://github.com/nix-community/home-manager/blob/2dc090fd7f7e0a76cff87c0e68ae15118e60e327/modules/services/window-managers/i3-sway/sway.nix#L310

In the way i3-config is generated tests require these new-lines and also I had to change my IDE to not trim trailing spaces. I would like to refactor the i3-code according and also fix the required trailing spaces but I think a separate PR is better to not blow up this as it is the current way it's handled?

Reply:
I agree, it is reasonable to make the formatting change in a separate PR. I think you can follow what I did in Sway (https://github.com/nix-community/home-manager/blob/master/modules/services/window-managers/i3-sway/sway.nix#L298)

Reply:
Yeah, that's what I meant I just referenced the implementation of your way as example :)
Then I will follow-up with this and mark the lines as resolved? Is there anything todo except rebase+squash?

Reply:
Yes, go ahead and mark the lines as resolved, and create an issue to keep track of the fix.

Reply:
Done #2030